README.md

```diff
@@ -131,9 +131,9 @@ Usage: odbapi2json.py [-h] [-c] [-q] input_odbapi output_temp > output.json
 
 ## chem
 
-The chem converters include all converter scripts for arosols and related chemistry variables. Currently only one Python script, `viirs_aod2ioda.py`, is used to convert the native netCDF format for observations of optical depth from VIIRS AOD550 to IODA netCDF format. Note that it takes only AOD550 explicitly and does not take the 11 AOD channels from VIIRS. The converter uses the following format to execute:
+The chem converters include all converter scripts for aerosols and related chemistry variables. Currently only one Python script, `viirs_aod2ioda.py`, is used to convert the native netCDF format for observations of optical depth from VIIRS AOD550 to IODA netCDF format. Note that it takes only AOD550 explicitly and does not take the 11 AOD channels from VIIRS. The converter uses the following format to execute:
  
 ```
 Usage: <converter.py> -i INPUT_FILE(S) -o OUTPUT_FILE -m nesdis -k maskout -t 0.0
 ```
-
+For method option (-m) of bias and uncertainty calculation (default/nesdis), deafult means to set bias and uncertainty as 0.0 and nesdis meansto use NESDIS bias and uncertainty calculation method. For maskout option (-k) default/maskout, default means to keep all missing values and maskout means to mask all missing values out. For thinning option, it value is within 0.0 and 1.0 depending how much data will be thinned, and 0.0 means without any thining.    
```

Comment:
"For thinning option, it value is " should probably say "For thinning option, the value should be " within 0.0 and 1.0...